src/main/java/com/enderio/base/common/blockentity/IWrenchable.java

```diff
@@ -0,0 +1,9 @@
+package com.enderio.base.common.blockentity;
+
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.item.context.UseOnContext;
+import net.minecraftforge.common.extensions.IForgeBlockEntity;
+
+public interface IWrenchable extends IForgeBlockEntity {
```

Comment:
While I think an interface is fine (might also just be able to live as a method in machineblock), why extend IForgeBlockEntity?

Reply:
Ahhh, you just mean that leaving it floating about extending nothing I gather? I think I used a method in a block entity related class at some point during one of the many iterations while trying to get everything to work. Probably mostly legacy, but pretty useful for indicating that you only should be able to use the wrench in the configuration other than wrenching on block entities and that was the most suitable interface that BlockEntity was implementing. But We could absolutely change this!